src/routes/+page.svelte

```diff
@@ -8,6 +16,10 @@
 <h1>Welcome to SvelteKit</h1>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
 
+<button class="button" on:click={viewSomeCampaign}>
+    Click me to view a campaign
```

Comment:
"View Campaign" would be more succinct

Reply:
It's shoddy, cuz it's extremely temporary.  It's the "I proved that it works!" button.  We can clean it up, or switch to a nav element of some kind, when we start displaying data.